noether_filtering/cloud_filter_plugins.xml

```diff
@@ -0,0 +1,129 @@
+<library path="lib/libnoether_filtering_filter_plugins">
+  <!-- pcl::PointXYZ -->
+  <class name="noether_filtering::cloud::VoxelGridFilter&lt;pcl::PointXYZ>" type="noether_filtering::cloud::VoxelGridFilter&lt;pcl::PointXYZ>"
+    base_class_type="noether_filtering::FilterBase&lt;boost::shared_ptr&lt;pcl::PointCloud&lt;pcl::PointXYZ> > >">
+    <description>
+      Voxel grid filter for PointXYZ type
+    </description>
+  </class>
+  <class name="noether_filtering::cloud::StatisticalOutlierFilter&lt;pcl::PointXYZ>" type="noether_filtering::cloud::StatisticalOutlierFilter&lt;pcl::PointXYZ>"
+    base_class_type="noether_filtering::FilterBase&lt;boost::shared_ptr&lt;pcl::PointCloud&lt;pcl::PointXYZ> > >">
```

Comment:
I'm not sure that the templated syntax works in the plugin.xml, have you been able to test this?

Reply:
Yes, I tested on my machine and another this week and can confirm the replacement of `<` with `&lt;` does work correctly (despite being really ugly)